Gemfile

```diff
@@ -13,7 +13,7 @@ gem "scss_lint"
 
 group :development do
   gem "rspec", "~> 3.5"
-  gem "capybara", "~> 2.11"
+  gem 'capybara', '~> 2.13'
```

Comment:
Need double quotes here. FYI, we use double quotes across the board for Ruby in all our projects.